src/client/components/RegistrationModal.js

```diff
@@ -4,6 +4,8 @@ import { useHistory, useLocation } from 'react-router-dom';
 import Button from './Button';
 import Modal from './Modal';
 
+import '../style/LoginModal.css';
```

Comment:
Normally, each component should have its own css file. If there are styles specific to `RegistrationModal`, I suggest we move them to a seperate css file. For those common styles (used by multiple components), we can move them `index.css` for easy maintenance.

Reply:
Created a separate RegistrationModal.css, cleaned up Login.css
Still, common/standard style across all pages is something that we should use.